provision/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+set -x
+
+# Mesosphere repo for Mesos and Marathon
+apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF
+echo "deb http://repos.mesosphere.io/debian jessie main" > /etc/apt/sources.list.d/mesosphere.list
+
+# Docker repo
+apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+echo "deb http://apt.dockerproject.org/repo debian-jessie main" > /etc/apt/sources.list.d/docker.list
+
+# Backports for OpenJDK 8 for Marathon 0.11+
+echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
+
+# Upgrade the system
+apt-get update
+apt-get upgrade -y
+
+# Install python things
+apt-get install -y \
+    python2.7 \
+    python-dev \
+    python-pip \
+    python-virtualenv
+
+# Tell dpkg not to overwrite our config files when installing mesos, marathon and supervisor
+apt-get install -y -o Dpkg::Options::="--force-confold" \
+    marathon \
+    mesos \
+    supervisor
+
+# Install docker and nginx
+apt-get install -y \
+    docker-engine \
+    nginx \
+    unzip
+
+# Curl and jq are useful to have
+apt-get install -y \
+    curl \
+    jq
+
+# Install consular
+pip install "pyasn1>=0.1.8"
```

Comment:
Just checked.. still required. I reckon the solution is to install/run consular in a virtualenv.